app/src/ui/app.tsx

```diff
@@ -1190,6 +1216,61 @@ export class App extends React.Component<IAppProps, IAppState> {
     })
   }
 
+  private openPullRequest() {
+    const selection = this.state.selectedState
+
+    if (!selection || selection.type !== SelectionType.Repository) {
+      return
+    }
+
+    const tip = selection.state.branchesState.tip
+
+    if (tip.kind !== TipState.Valid) {
+      return
+    }
+
+    const dispatcher = this.props.dispatcher
+    const repository = selection.repository
+    const branch = tip.branch
+    const aheadBehind = selection.state.aheadBehind
+
+    if (!aheadBehind) {
+      //Pubilish branch
```

Comment:
I'm not sure this comment and the one in the next branch are really adding much.